catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -456,6 +456,8 @@
                             !org.hibernate.*,
                             !weblogic.jdbc.*,
                             !org.abego.treelayout.*,
+                            <!-- TODO - verify why excluding bouncycastle was needed to bundle:watch -->
+                            !org.bouncycastle.crypto.*,
```

Comment:
❗️ Need to follow up on this.

Reply:
I wonder if it is because the bouncy castle packages are loaded into bundle 0. Then, when this guy tries to bundle watch, it may be identifying a dependency on bundle 0 and in affect tell bundle 0 to update 🤷‍♂️

Reply:
It worked without this before. I'm just surprised rearranging the blueprint would have that impact.

Reply:
Also, @bakejeyner wasn't bouncycastle removed?

Reply:
BC wasn't removed, but it was updated recently. And I didn't know that bundle watch only started failing after the refactor. That's some weird juju.

Reply:
Nope, scratch that. I'm misremembering. We removed Keyczar for Tink. We still have boucycastle.